### PR TITLE
Add OpenSea Link, fix Polyscan link, add new favicon

### PR DESCRIPTION
**Pull request by Tudmotu**

### CODE REVIEW

File: src/lib/MintButton.svelte [hunk 1]
```diff
@@ -32,7 +32,7 @@
         {
           chainId: ethers.utils.hexValue(parseInt($globalState.neededChainId)),
           rpcUrls: ["https://rpc-mainnet.matic.network/"],
-          chainName: "Matic Testnet (Mumbai)",
+          chainName: "Polygon",
```
Comment (royosherove) [inline src/lib/MintButton.svelte]: I have fixed this separately so remove from PR

File: src/routes/index.svelte [hunk 2]
```diff
@@ -27,6 +27,6 @@ import LinksBoard from "$lib/LinksBoard.svelte";
         Credits: Yotam Bar-On, ChagitR, Royo
       </div>
       <a href='https://github.com/royosherove/afikoman' class="text-blue-400 ">github</a>
-      <a href='https://mumbai.polygonscan.com/address/0x9dA9d1c92Bf2aC5cd21925242Ff854932465df5C#code' class="text-blue-400 ">contract</a>
+      <a href='https://polygonscan.com/address/0xf3315750a88AB66A730C6813B43FC03248baC7D3#code' class="text-blue-400 ">contract</a>
```
Comment (royosherove) [inline src/routes/index.svelte]: Maybe we can add this to stores.ts to be more generic. up to you.

**Review by royosherove — CHANGES_REQUESTED** [review]

File: src/routes/index.svelte (hunk 2 above)
Comment (Tudmotu) [reply src/routes/index.svelte]: Changed it to use contractAddress from store

File: src/lib/MintButton.svelte (hunk 1 above)
Comment (Tudmotu) [reply src/lib/MintButton.svelte]: Fixed

**Review by royosherove — APPROVED** [review]

**Pull request merged**